src/SCEA.xml

```diff
@@ -0,0 +1,151 @@
+             "Contributions"). Per the terms of this License You are free to modify the Software and those
+             modifications would belong to You. You may however wish to donate Your Contributions to SCEA
+             for consideration for inclusion into the Software. For the avoidance of doubt, if You elect to
+             send Your Contributions to SCEA, You are doing so voluntarily and are giving the Contributions
+             to SCEA and its parent company Sony Computer Entertainment, Inc., free of charge, to use,
+             modify or distribute in any form or in any manner. SCEA acknowledges that if You make a
+             donation of Your Contributions to SCEA, such Contributions shall not exclusively belong to
+             SCEA or its parent company and such donation shall not be to Your exclusion. SCEA, in its sole
+             discretion, shall determine whether or not to include Your donated Contributions into the
+             Software, in whole, in part, or as modified by SCEA. Should SCEA elect to include any such
+             Contributions into the Software, it shall do so at its own risk and may elect to give credit
+             or special thanks to any such contributors in the attached copyright notice. However, if any
+             of Your contributions are included into the Software, they will become part of the Software
+             and will be distributed under the terms and conditions of this License. Further, if Your
+             donated Contributions are integrated into the Software then Sony Computer Entertainment, Inc.
+             shall become the copyright owner of the Software now containing Your contributions and SCEA
+             would be the Licensor.</p>
+        </li>
+        <li>
+          <b>5.</b>
+          <p>Redistribution in Source Form</p>
+          <p>You may redistribute copies of the Software, modifications or derivatives thereof in Source Code
+             Form, provided that You:</p>
+        </li>
+      </list>
+      <list>
+        <li>
+          <b>a.</b>
+          <p>Include a copy of this License and any copyright notices with source</p>
+        </li>
+        <li>
+          <b>b.</b>
+          <p>Identify modifications if any were made to the Software</p>
+        </li>
+        <li>
+          <b>c.</b>
+          <p>Include a copy of all documentation accompanying the Software and modifications made by You</p>
+        </li>
+        <li>
+          <b>6.</b>
+          <p>Redistribution in Object Form</p>
+          <p>If You redistribute copies of the Software, modifications or derivatives thereof in Object Form
+             only (as incorporated into finished goods, i.e. end user applications) then You will not have
+             a duty to include any copies of the code, this License, copyright notices, other attributions
+             or documentation.</p>
+        </li>
+        <li>
```

Comment:
Bullet 6) is part of the previous list not list a), b)... Needs to be outdented,